From d8c3399e1af199f0998ad4633b4c736a2a083bcd Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Fri, 26 Oct 2012 12:46:54 -0700
Subject: [PATCH 6/9] invoke catkin at the end to deal with existing macros

---
 CMakeLists.txt |    7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 479a290..61e71ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,6 @@ if(COMMAND cmake_policy)
 endif(COMMAND cmake_policy)
 
 project(flann)
-find_package(catkin)
-catkin_package()
 string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
 
 include(${PROJECT_SOURCE_DIR}/cmake/flann_utils.cmake)
@@ -194,3 +192,8 @@ message(STATUS "Building python bindings: ${BUILD_PYTHON_BINDINGS}")
 message(STATUS "Building matlab bindings: ${BUILD_MATLAB_BINDINGS}")
 message(STATUS "Building CUDA library: ${BUILD_CUDA_LIB}")
 message(STATUS "Using MPI support: ${USE_MPI}")
+
+
+find_package(catkin)
+catkin_package()
+
-- 
1.7.10.2 (Apple Git-33)

